src/components/Button/Button.js

```diff
@@ -1,27 +1,43 @@
 // packages
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
-const Button = ({clickData, color, onClick, text, type, width}) => {
-  const styles = {
-    width,
-    color
-  };
+const Button = ({backgroundColor, borderRadius, clickData, color, onClick, text, type, width}) => (
+  <TaskRemove
+    backgroundColor={backgroundColor}
+    borderRadius={borderRadius}
+    color={color}
+    width={width}
+    type={type}
+    onClick={() => onClick(clickData)}
+  >{text}</TaskRemove>
+);
 
-  return (
-    <button type={type} style={styles} onClick={() => onClick(clickData)}>{text}</button>
-  );
-};
+const TaskRemove = styled.button`
+  background-color: ${props => props.backgroundColor};
+  border-radius: ${props => props.borderRadius}px;
+  color: ${props => props.color};
+  width: ${props => props.width}px;
+`;
 
 Button.propTypes = {
   /**
+   * Colour of background
+   */
+  backgroundColor: PropTypes.string,
```

Comment:
This is not required but yet in line 18 you are assigning its value in CSS property. What happens if its value is not provided?